### ISSUE TITLE
Add grand total to bottom of table+HTML output

### ISSUE THREAD

**Issue by alikhajeh1** [infracost/infracost#734]
Currently we show the per-project total. Should we show the total for all projects at the bottom of the table and the new HTML (v0.9) output formats? It's already in the JSON output but marked as deprecated, depending on what we decide with this feature we can rename that param so it's consistent with the table+html outputs.

Should we call this Grand Total or Run Total (each CLI run)?

<details>
  <summary>Example table format</summary>

```
Project: examples/terraform

 Name                                                        Monthly Qty  Unit                  Monthly Cost

 aws_instance.web_app
 ├─ Instance usage (Linux/UNIX, on-demand, m5.4xlarge)               730  hours                      $560.64
 ├─ root_block_device
 │  └─ Storage (general purpose SSD, gp2)                             50  GB-months                    $5.00
 └─ ebs_block_device[0]
    ├─ Storage (provisioned IOPS SSD, io1)                         1,000  GB-months                  $125.00
    └─ Provisioned IOPS                                              800  IOPS-months                 $52.00

 aws_lambda_function.hello_world
 ├─ Requests                                            Cost depends on usage: $0.20 per 1M requests
 └─ Duration                                            Cost depends on usage: $0.0000166667 per GB-seconds

 PROJECT TOTAL                                                                                       $742.64

----------------------------------
Project: examples/terraform

 Name                                                        Monthly Qty  Unit                  Monthly Cost

 aws_data_transfer.my_region
 ├─ Intra-region data transfer                                     1,000  GB                          $10.00
 ├─ Outbound data transfer to Internet                             5,000  GB                         $425.00
 ├─ Outbound data transfer to US East regions                        500  GB                           $5.00
 └─ Outbound data transfer to other regions                          750  GB                          $15.00

 aws_instance.web_app
 ├─ Instance usage (Linux/UNIX, on-demand, m5.4xlarge)               730  hours                      $560.64
 ├─ root_block_device
 │  └─ Storage (general purpose SSD, gp2)                             50  GB-months                    $5.00
 └─ ebs_block_device[0]
    ├─ Storage (provisioned IOPS SSD, io1)                         1,000  GB-months                  $125.00
    └─ Provisioned IOPS                                              800  IOPS-months                 $52.00

 aws_lambda_function.hello_world
 ├─ Requests                                            Cost depends on usage: $0.20 per 1M requests
 └─ Duration                                            Cost depends on usage: $0.0000166667 per GB-seconds

 PROJECT TOTAL                                                                                     $1,197.64

 GRAND TOTAL                                                                                       $1,940.28
----------------------------------
To estimate usage-based resources use --usage-file, see https://infracost.io/usage-file
```
</details>

**Comment by aliscott** [comment]
1. Should we use grand total or overall total? When we were showing resource totals before we used the terminology "Overall total", but I don't have a strong preference.
2. Should we add this to `infracost diff` as well? If we do then I can see the output getting too cluttered, e.g:
```
...
----------------------------------
Project: examples/terraform

+ aws_instance.web_app
  +$743

    + Instance usage (Linux/UNIX, on-demand, m5.4xlarge)
      +$561

Monthly cost change for examples/terraform:
Amount:  +$443 ($200 -> $743)
Percent: +36%

Grand total monthly cost change:
Amount:  +$515 ($600 -> $1115)
Percent: +54%
```
3. Is there any point showing the project totals if there's only one project?

**Comment by alikhajeh1** [comment]
2. I don't think it's useful to see the overall/grand total in the diff output.
3. Nope, best to avoid that as it's confusing seeing 2 totals.

**Comment by alikhajeh1** [comment]
released in v0.9.0
